### DISCUSSION TITLE
InteractableBed.OnBedClaimRequested

### DISCUSSION THREAD

**Discussion by sunnamed434** [SmartlyDressedGames/Unturned-3.x-Community#3761] [Ideas]
i.e signature of the method would look like OnBedClaimRequested(InteractableBed bed, BarricadeDrop barricadeDrop, Player player, ref bool shoudAllow)

**Comment by GazziFX** [comment]
Remove `x, y, plant, region` or replace with `BarricadeDrop` as it become obsolete in most methods

**Reply by sunnamed434** [reply]
done
